custom.ts

```diff
@@ -733,34 +741,34 @@ namespace gigglebot {
      */
     //% blockId="gigglebot_show_voltage" block="show battery voltage (mv)"
     //% advanced=true
-    export function show_voltage() {
-        let voltage = get_voltage()
+    export function voltageShow() {
+        let voltage = voltageRead()
         basic.showNumber(voltage)
     }
 
     //% blockId="gigglebot_get_firmware" block="firmware version number"
     //% advanced=true
-    export function get_firmware(): number {
+    export function firmwareRead(): number {
         /**
          * TODO: describe your function here
          * @param value describe value here, eg: 5
          */
         let buf = pins.createBuffer(1)
-        buf.setNumber(NumberFormat.UInt8BE, 0, I2C_Commands.GET_FIRMWARE_VERSION)
+        buf.setNumber(NumberFormat.UInt8BE, 0, gigglebotI2CCommands.GET_FIRMWARE_VERSION)
         pins.i2cWriteBuffer(ADDR, buf)
         let val = pins.i2cReadBuffer(ADDR, 2)
         return val.getNumber(NumberFormat.UInt16BE, 0);
     }
 
     //% blockId="gigglebot_get_voltage" block="battery voltage (mv)"
     //% advanced=true
-    export function get_voltage(): number {
+    export function voltageRead(): number {
```

Comment:
Since the newer version of HW (and corresponding FW) will have two voltages (battery and rail), should this be something like "voltageRail", or "voltage5v"? The complement would be "voltageBattery".

Reply:
fixed